### PR TITLE
Add command to set velocity and acceleration of powered rides

### PR DESCRIPTION
**Pull request by X123M3-256**
This implements the same functionality as #3855, but I messed up that branch trying to update it to work with the latest code.

### CODE REVIEW

File: src/interface/console.c [hunk 1]
```diff
@@ -459,47 +490,52 @@ static int cc_rides(const utf8 **argv, int argc)
...
-				} else {
-					gGameCommandErrorTitle = STR_CANT_CHANGE_OPERATING_MODE;
-					int res = game_do_command(0, (type << 8) | 1, 0, (RIDE_SETTING_RIDE_TYPE << 8) | ride_index, GAME_COMMAND_SET_RIDE_SETTING, 0, 0);
-					if (res == MONEY32_UNDEFINED) {
-						console_printf("That didn't work");
-					}
+				gGameCommandErrorTitle = STR_CANT_CHANGE_OPERATING_MODE;
+				int res = game_do_command(0, (value << 8) | 1, 0, (RIDE_SETTING_RIDE_TYPE << 8) | ride_index, GAME_COMMAND_SET_RIDE_SETTING, 0, 0);
+				if (res == MONEY32_UNDEFINED) {
+					console_printf("That didn't work");
```
Comment (duncanspumpkin) [inline src/interface/console.c]: This isn't the greatest error message and doesn't really match what other error messages are.

**Comment by duncanspumpkin** [comment]
Looks good. Can someone test just to double check. Would an option for setting this for an individual vehicle be useful ever?

File: src/interface/console.c (hunk 1 above)
Comment (X123M3-256) [inline src/interface/console.c]: That's not my error message - that's what's already in develop. I didn't modify the existing cheat, I just added the new one. I agree that that error message isn't great. I could change it if you want, but I'm not sure what too - I don't know what that return value signifies.

Comment (Gymnasiast) [inline src/interface/console.c]: "Could not set the velocity or acceleration for this vehicle" or something along these lines.

Comment (X123M3-256) [inline src/interface/console.c]: @Gymnasiast That error message is for the existing ride type cheat, not the one I added. The one I added

Comment (Gymnasiast) [inline src/interface/console.c]: Well, in that case please adjust the message accordingly.

**Comment by janisozaur** [comment]
@X123M3-256 any chance you could use `game_do_command` in your code? There's already `GAME_COMMAND_SET_RIDE_SETTING` you could extend.

**Comment by X123M3-256** [comment]
I'm not exactly sure how I would go about extending it. I could add it as a new ride setting but it isn't really, because it's not a single number. This value is set independently for every vehicle, the cheat just sets everything to the same value.

**Comment by Gymnasiast** [comment]
Could you take a look at the code style? It's a recurring thing in your PRs.

File: src/interface/console.c [hunk 2]
```diff
@@ -459,47 +490,53 @@ static int cc_rides(const utf8 **argv, int argc)
 			if (argc < 4) {
 				console_printf("rides set type <ride id> <ride type>");
 				console_printf("rides set friction <ride id> <friction value>");
+				console_printf("rides set powered_velocity <ride id> <velocity> [acceleration]");
 				return 0;
 			}
+			int optional=-1;
```
Comment (Gymnasiast) [inline src/interface/console.c]: Needs spaces around the operator.

**Comment by X123M3-256** [comment]
I added two spaces and now it says there's a merge conflict? How do I fix this?

**Comment by Gymnasiast** [comment]
The merge conflict isn't because of your spaces, but because of a PR that has moved the source files which just got merged. You need to run (assuming you have this remote as `upstream`):
```
git fetch upstream
git rebase -i upstream/develop
git push -f
```

**Comment by Gymnasiast** [comment]
Could you rebase this again?

**Comment by Gymnasiast** [comment]
I mucked up the rebase :/. Could you create a new PR?

**Comment by IntelOrca** [comment]
It doesn't look like this is getting resolved, so closing.

**Comment by Gymnasiast** [comment]
Sounds fair. I'll message X7 again to see if he wants another go.

**Pull request closed without merging**